packages/front-end/components/Experiment/Results.tsx

```diff
@@ -186,6 +190,8 @@ const Results: FC<{
           setVariationFilter={(v: number[]) => setVariationFilter?.(v)}
           baselineRow={baselineRow}
           setBaselineRow={(b: number) => setBaselineRow?.(b)}
+          differenceType={differenceType}
+          setDifferenceType={(d: DifferenceType) => setDifferenceType?.(d)}
```

Comment:
Yeah I don't think there's a good reason. I was following the other behavior because I think this is missing in some of the Results_old components.

I'll change this and test it out, following the places where it's possibly missing.

Reply:
Removed it and cleaned up the logic in AnalysisSettingsSummary because it should never be undefined there.